examples/preprocessing/plot_muscle_detection.py

```diff
@@ -0,0 +1,84 @@
+"""
+===========================
+Annotate muscle artifacts
+===========================
+
+Muscle contractions produce high frequency activity that can mask brain signal
+of interest. Muscle artifacts can be produced when clenching the jaw,
+swallowing, or twitching a head muscle. Muscle artifacts are most noticeable
+in the range of 110-140 Hz.
+
+This example uses :func:`~mne.preprocessing.annotate_muscle_zscore` to annotate
+segments where muscle activity is likely present. This is done by band-pass
+filtering the data in the 110-140 Hz range. Then, the envelope is taken using
+the hilbert analytical signal to only consider the absolute amplitude and not
+the phase of the high frequency signal. The envelope is z-scored and summed
+across channels and divided by the square root of the number of channels.
+Because muscle artifacts last several hundred milliseconds, we then low-pass
+filter the averaged z-scores at 4 Hz, to remove transient peaks. Segments above
```

Comment:
I'm not sure all this description should be here. You are in fact describing the internals of the `annotate_muscle_zscore` here, so statements like "we then low-pass filter the ..." may be slightly confusing. I'd either simplify the description (and refer to `annotate_muscle_zscore` docs) or make it clearer that it is not "we" that are doing it in the example, but the function does that internally.

Reply:
the "we then low-pass filter the ..." part is not mine 😃I must have just accepted a commit. I was using a passive voice to describe it. I will change it.
I personally like examples that are didactic and avoids me having to look at the code to know what the example does. I am fine if you want to push a commit with another explanation.